src/modals/BottomModal.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { motion } from 'framer-motion'
+import styled from 'styled-components'
+
+import { fastTransition } from '@/animations'
+import Scrollbar from '@/components/Scrollbar'
+import useFocusOnMount from '@/hooks/useFocusOnMount'
+
+import ModalContainer, { ModalContainerProps } from './ModalContainer'
+
+interface BottomModalProps extends ModalContainerProps {
+  label: string
+  contentHeight?: number
+  className?: string
+}
+
+const BottomModal = ({ onClose, children, label, contentHeight, className }: BottomModalProps) => {
+  const elRef = useFocusOnMount<HTMLDivElement>()
```

Comment:
This is probably a remnant from an attempt to compute the height manually, and could be removed for now :)

Reply:
Actually not! This is to move the focus to the modal for 2 reasons:
1. accessibility (screen readers)
2. so that the ESC key press is captured by the modal

Reply:
Ah ok! Perfect.